code/game/jobs/access.dm

```diff
@@ -224,21 +233,21 @@ var/global/list/priv_region_access
 
 // Gets the ID card of a mob, but will not check types in the exceptions list
 /mob/living/carbon/human/GetIdCard(exceptions = null)
-	var/list/id_cards = get_held_items()
-	LAZYDISTINCTADD(id_cards, wear_id)
-	for(var/obj/item/I in id_cards)
-		if(is_type_in_list(I, exceptions))
+	return LAZYACCESS(GetIdCards(exceptions), 1)
+
+/mob/living/carbon/human/GetIdCards(exceptions = null)
+	. = ..()
+	var/list/candidates = get_held_items()
+	LAZYDISTINCTADD(candidates, wear_id)
+	for(var/atom/movable/candidate in candidates)
+		if(!candidate || is_type_in_list(candidate, exceptions))
```

Comment:
Won't this prevent multiple access cards being tracked?

Reply:
I don't believe it will? It just skips nulls and excluded types. For what it's worth, I'm not a big fan of the exceptions argument, but existing things use it, so.

Reply:
Oh I somehow completely misread exceptions as being a list of previously observed types. Disregard.